Crane/gradle/libs.versions.toml

```diff
@@ -6,14 +6,14 @@
 accompanist = "0.28.0"
 androidGradlePlugin = "7.3.1"
 androidx-activity-compose = "1.6.1"
-androidx-appcompat = "1.5.1"
+androidx-appcompat = "1.6.0"
 androidx-benchmark = "1.1.0"
 androidx-benchmark-junit4 = "1.1.0-beta04"
-androidx-compose-bom = "2022.12.00"
+androidx-compose-bom = "2023.01.00"
 androidx-constraintlayout = "1.0.1"
 androidx-corektx = "1.9.0"
-androidx-lifecycle-compose = "2.5.1"
-androidx-lifecycle-runtime-compose = "2.6.0-alpha03"
+androidx-lifecycle-compose = "2.6.0-alpha04"
```

Comment:
Just a double check that we're okay updating lifecycle-compose to an alpha version. I guess it is grouped with lifecycle-runtime-compose?

Reply:
As we're using alphas already for runtime, I personally don't see a problem?